app/javascript/legacy/nonprofits/donate/followup-step.js

```diff
@@ -14,15 +14,15 @@ function view(state) {
       h('a.button--small.facebook.u-width--full.share-button', {
         props: {
           target: '_blank'
-        , href: 'https://www.facebook.com/dialog/feed?app_id='+app.facebook_app_id +"&display=popup&caption=" + encodeURIComponent(app.campaign.name || app.nonprofit.name) + "&link="+window.location.href + "?/utm_source=facebook&utm_medium=organic_social&utm_campaign=" + encodeURIComponent(app.campaign.name)
+        , href: 'https://www.facebook.com/dialog/feed?app_id='+app.facebook_app_id +"&display=popup&caption=" + encodeURIComponent(app.campaign.name || app.nonprofit.name) + "&link="+window.location.href 
         }
       }, [h('i.fa.fa-facebook-square'), ` ${I18n.t('nonprofits.donate.followup.share.facebook')}`] )
     ])
   , h('div.u-inlineBlock.u-marginLeft--10.u-marginBottom--20', [
       h('a.button--small.twitter.u-width--full', {
         props: {
           target: '_blank'
-        , href: "https://twitter.com/intent/tweet?url="+window.location.href+"&via=CommitChange&text=Join me in supporting:" + (app.campaign.name || app.nonprofit.name) + "?/utm_source=twitter&utm_medium=organic_social&utm_campaign=" + encodeURIComponent(app.campaign.name)
+        , href: "https://twitter.com/intent/tweet?url="+window.location.href+"&via=CommitChange&text=Join me in supporting:" + (app.campaign.name || app.nonprofit.name)
```

Comment:
Is there a reason you moved away from  having the `utm_medium` and `utm_campaign` here? I think it would probably be useful.